dev-tex/biber/biber-2.16.ebuild

```diff
@@ -67,3 +67,10 @@ DEPEND="${RDEPEND}
 			dev-perl/Test-Differences )"
 
 mydoc="doc/biber.tex"
+
+src_prepare() {
+	#disable 64-bit only Tests on non 64-bit archs
+	use amd64 || use arm64 || eapply "${FILESDIR}/${P}-disable-64bit-only-tests.patch"
```

Comment:
I wonder if there's a cleaner way to do this (detect it), but hey, works for now.